src/tests/ftest/datamover/posix_preserve_props.py

```diff
@@ -77,7 +77,7 @@ def run_dm_preserve_props(self, tool, cont_type, api):
         # move data from POSIX path to DAOS and read container properties in hdf5 file to cont2
         result = self.run_datamover(
             self.test_id + " posix to cont2 (empty cont)",
-            "POSIX", posix_path, None, None,
+            "POSIX", posix_path + "/" + self.test_file, None, None,
```

Comment:
`posix_path` is created above with `posix_path = join(self.new_posix_test_path(), self.test_file)`
So isn't this wrong?

Reply:
@daltonbohning well this was how it was before and is passing because of how ior is run to read/verify the testfile. I think it is confusing yes I can update.

Reply:
@daltonbohning what this is doing is creating a testFile directory on the first copy and it is working because the actual file ior verifies is at <posix_path>/testFile/testFile which is working but weird. I updated it to make it less confusing.

Reply:
Ah, I see. Thanks! :)